lighthouse-cli/test/cli/run-test.js

```diff
@@ -58,8 +58,21 @@ describe('Parsing --chrome-flags', () => {
     assert.deepStrictEqual([], parseChromeFlags());
   });
 
+  it('keeps --no-flags untouched, #3003', () => {
+    assert.deepStrictEqual(['--no-sandbox'], parseChromeFlags('--no-sandbox'));
```

Comment:
any reason these are backwards from the usual actual/expected order?

Reply:
I didn't notice that the first time around here, but I was just wondering the same haha

Reply:
yes will happily swap them. it was breaking my brain too.
i blame the other guy.